src/style.css

```
@@ -0,0 +1,118 @@
+
+body {
+  margin: 0;
+}
```

put font family in the body selector and add fallback fonts
```suggestion
body {
  margin: 0;
  
  font-family: Roboto,  Arial, sans-serif;
}
```